sympy/physics/vector/point.py

```diff
@@ -471,6 +471,27 @@ def v2pt_theory(self, otherpoint, outframe, fixedframe):
         self.set_vel(outframe, v + (omega ^ dist))
         return self.vel(outframe)
 
+    def _calc_vel(self, frame, rel_pos):
+        if len(self._pos_dict) == 1:
+            return False
+        for p, p_rel_pos in self._pos_dict.items():
+            if p_rel_pos == -rel_pos:
+                continue
+            try:
+                p_abs_vel = p._vel_dict[frame]
+            except KeyError:
+                if not p_rel_pos.dt(frame) == 0:
```

Comment:
This `if` doesn't make sense to me - if I've specified a distance as constant, why would I not expect a relative velocity of 0?

Reply:
This if is just for checking frame,  that the distance is specified using appropriate frame or not